web-api/src/persistence/postgres/utils/seed/fixtures/messages.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable @miovision/disallow-date/no-new-date */
+export const messages = [
```

Comment:
Can we make these seed messages have a type so that we know if they end up getting out of date?

Reply:
it has a type, seed will complain if it's not the correct type, it's just assumed, but will make it explicit.